src/main/steps/common/task-list/components/notification-banner/definitions.ts

```diff
@@ -127,4 +128,5 @@ export enum NotificationID {
   DA_RESPONDENT_BANNER = 'DN3_SOA_RESPONDENT',
   ORDER_NON_PERSONAL_SERVICE = 'CRNF2_APPLICANT_RESPONDENT',
   ORDER_PERSONAL_SERVICE = 'CRNF3_PERS_SERV_APPLICANT',
+  DA_APPLICATION_SERVED_BY_COURT_PERSONAL_NONPERSONAL_SERVICE = 'DN1_SOA_PERSONAL_NON_PERSONAL_APPLICANT',
```

Comment:
Rename to APPLICATION_SERVED_BY_COURT_PERSONAL_NONPERSONAL_SERVICE_TO_DA_APPLICANT and pls add it below CA notificationID so that it is easy to understand when grouped